README.md

```diff
@@ -2,18 +2,20 @@
 A comprehensive demonstration Ansible-based solution to quickly build and deploy an application in a public cloud using F5 and NGINX technology
 
 ## Featuring
-- Grafana
-- Graphite
-- StatsD
-- ElasticSearch
+- [F5 BIG-IP](https://www.f5.com/products/big-ip-services)
+- [Grafana](https://grafana.com/)
+- [Graphite](https://graphiteapp.org/)
+- [StatsD](https://github.com/statsd/statsd)
+- [ElasticSearch](https://www.elastic.co/products/elastic-stack)
 - [Juice Shop app](https://hub.docker.com/r/bkimminich/juice-shop/)
+- [OWASP Zed Attack Proxy (ZAP)](https://github.com/zaproxy/zaproxy)
 
 ## Dependencies
-- Ansible 2.8
-- Inspec 4.7+
+- [RedHat Ansible](https://www.ansible.com/) 2.8
+- [Chef Inspec](https://www.inspec.io/) 4.7+
 
 ## Prerequisites
-- This demo assumes an Ansible and Docker host of Ubuntu 18.04 (Bionic)
+- This demo assumes an Ansible and [Docker](https://www.docker.com/) host running on [Ubuntu](https://ubuntu.com/) release 18.04 (Bionic)
```

Comment:
I thought that the point was for the Docker guest to be running Ubuntu 18.04. If that's not the case you can resolve this issue.

Reply:
No, the host needs to be 18.04. The guests are of varying stacks (Alipne linux, etc)